app/src/main/res/layout/card_events_item.xml

```diff
@@ -1,76 +1,59 @@
 <android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    style="@style/StandardCard">
+    style="@style/StandardCard"
+    app:cardUseCompatPadding="true">
 
     <LinearLayout
         android:id="@+id/card_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="horizontal">
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
 
         <ImageView
             android:id="@+id/events_img"
-            android:layout_width="160dp"
-            android:layout_height="240dp"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:adjustViewBounds="true"
+            android:maxHeight="200dp"
+            android:minHeight="50dp"
             android:scaleType="centerCrop"
             android:src="@drawable/mvv_entire_net_icon" />
 
-        <RelativeLayout
-            android:layout_width="fill_parent"
-            android:layout_height="fill_parent"
-            android:orientation="vertical">
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:padding="@dimen/material_default_padding">
 
             <TextView
                 android:id="@+id/events_title"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_above="@+id/linearLayout"
-                android:layout_alignParentEnd="true"
-                android:layout_alignParentStart="true"
-                android:layout_alignParentTop="true"
                 android:layout_margin="10dp"
```

Comment:
Is the margin for the title set on purpose? Do we need it?

Reply:
Since i changed the layout of cards. I think it is useless